app/controlls/List.js

```diff
@@ -152,9 +152,31 @@ SDL.List = Em.ContainerView.extend({
                 classNames: 'list-item',
 
                 classNameBindings: [
-                  'this.voiceOver'
+                  'this.voiceOver',
+                  'dayMode',
+                  'nightMode',
+                  'highLightedMode'
                 ],
 
+                dayMode:false,
+                nightMode:false,
+                highLightedMode:false,
+
+                setMode:function(mode) {
+                  this.set('dayMode',false);
+                  this.set('nightMode',false);
+                  this.set('highLightedMode',false);

```

Comment:
@YarikMamykin you can remove first three lines with set to `false` as the next three lines will overwrite the previous value anyway

Reply:
@AKalinich-Luxoft remove in https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/32ed6aec1f1bcf1414a9b0b4bd6237e48fbbe478